### PR TITLE
[TR] [PAM] PIM-4074: Remove an asset

### PR DESCRIPTION
**Pull request by willy-ahva**
A temporary fix has to be removed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/manager/attribute-manager.js [hunk 1]
```diff
@@ -124,6 +124,9 @@ define([
                         }
 
                         if ('pim_catalog_price_collection' === attribute.type) {
+                            if (null === newValue.data) { // TODO revert this temporary fix
```
Comment (jjanvier) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/manager/attribute-manager.js]: to revert

**Comment by jmleroux** [comment]
no phpspec ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/manager/attribute-manager.js (hunk 1 above)
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/manager/attribute-manager.js]: As long as this bug is not fixed, if I removed this temp fix behats will not pass.

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/manager/attribute-manager.js]: better skip the behat and create a card

**Pull request closed without merging**